rodan/jobs/helloworld/helloworld.py

```diff
@@ -29,6 +39,46 @@ def run_my_task(self, inputs, settings, outputs):
         outfile.close()
         return True
 
+class HelloWorld_1PortType_MultiPorts(RodanTask):
+    name = 'Hello World MultiTasks'
+    author = 'Silver'
+    description = "A demo of showing the multiports job functions, didn't add list type parsing"
+    settings = {}
+    enabled = True
+    category = "Test"
+    interactive = False
+
+    input_port_types = (
+        {
+            'name': 'Text input', 
+            'minimum': 0, 
+            'maximum': 10, 
+            'resource_types': ['text/plain']
+        },
+    )
+    output_port_types = (
+        {
+            'name': 'Text output', 
+            'minimum': 1, 
+            'maximum': 10, 
+            'resource_types': ['text/plain']
+        },
+    )
+
+    def run_my_task(self, inputs, settings, outputs):
+        for output_type in outputs:
+            for output in outputs[output_type]:
+                outfile = open(output["resource_path"], "w")
+                # Write input file infomation into the current 
+                # out file.
+                for input_type in inputs:
+                    for input in inputs[input_type]:
+                        infile = open(input["resource_path"], "r")
+                        outfile.write(infile.read() + "\n")
+                        infile.close()
```

Comment:
⚠️ Heads-up: this loop is unrelated to output and can surely be moved outside.

Reply:
I can extract an additional method of reading the input file